source/documentation/deploying_services/postgres.md

```diff
@@ -172,16 +172,16 @@ For more details about how the RDS backup system works, see [Amazon's DB Instanc
 
 ### Restoring a PostgreSQL service snapshot (experimental)
 
-As a tenant, you can create a copy of any existing PostgreSQL service instance using the latest snapshot of the RDS instance.
+You can create a copy of any existing PostgreSQL service instance using the latest snapshot of the RDS instance.
```

Comment:
I think the distinction is clear from a content perspective and that 'As a tenant' isn't needed.